linkedin_learning.py

```diff
@@ -113,13 +111,10 @@ def chapter_dir(course: Course, chapter: Chapter):
         data = await resp.json()
         course = build_course(data['elements'][0])
 
-        logging.info(f'[*] Access to {course.name} is {"GRANTED" if course.unlocked else "DENIED"}')

```

Comment:
what about locked course? just letting it fail on fetching? maybe worth printing something here

Reply:
I noticed the code was always logging "DENIED", like if `course.unlocked` was always false (or not defined anymore). And because it was always "DENIED", nothing was downloaded, that's why someone before me commented this block:
```
        #if not course.unlocked:
            # Nothing to do here
            #return
```
So I decided to just remove this wrong information from the log.
I don't have an example of locked course, so I can't fix what was probably working initially.

Reply:
Well then, let's leave it to the audience :)